tasks/pip.yml

```diff
@@ -113,7 +112,7 @@
 - name: Install extra python packages
   pip:
     name: "{{ openwisp2_extra_python_packages }}"
-    state: latest
+    state: forcereinstall
```

Comment:
why is this needed?

Reply:
Before `openwisp-network-topology` was released, I was getting an error swapper isn't installed because of `--no-deps` didn't allow dependencies to be installed. Hence, I removed `--no-deps`, then I realized that I was trying to install the same version of network-topology, but that wouldn't happen because it is already in `latest` (as version had not changed); I realized that a user may find themselves in the same problem trying to install their own version, so I changed it to `forceinstall`, which causes idempotence to fail, so I had to add `molecule-idempotence-notest`.

But after the release I am not able to reliably reproduce this problem with openwisp-utils. Hence, I am undoing all these changes. It's not specifically related to this PR and will stall this work needlessly.

But I think we should open a separate issue for this `--no-deps` problem.
Please let me know what do you think about it. :smile:

Reply:
@atb00ker yes ok, let's open another issue for this :+1: